Задания с 18.02.2022/Task2/Task2/BWT.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+public class BWT
+{
+    private static string leftShift(string line, int count)
+    {
+        return line.Substring(count) + line.Substring(0, count);
+    }
+
+    public static string Direct(string line, ref int endPosition)
```

Comment:
```suggestion
    public static string Direct(string line, out int endPosition)
```
Я, конечно, про это не рассказывал, но ref --- это входной-выходной параметр, а out --- только выходной (и компилятор проверяет, что внутри метода ему присваивают значение). А ещё лучше так:
```suggestion
    public static (string, int) Direct(string line)
```
--- вернуть сразу пару значений.